cards/src/main/resources/cards/custom/group3/set_1087/minion_tower_ophidian.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Tower Ophidian",
+  "baseManaCost": 5,
+  "type": "MINION",
+  "heroClass": "RUST",
+  "baseAttack": 2,
+  "baseHp": 6,
+  "rarity": "COMMON",
+  "race": "DRAGON",
+  "description": "Supremacy: Summon a copy of this minion.",
+  "trigger": {
+    "eventTrigger": {
+      "class": "SupremacyTrigger"
+    },
+    "spell": {
+      "class": "SupremacySpell",
+      "spell": {
+        "class": "SummonSpell",
+        "card": "minion_tower_ophidian"
```

Comment:
this should be `"target": "SELF"` instead of `"card...` because it's summoning a copy of itself, not another copy of a Tower Ophidian generally.